scripts/zones/Arrapago_Reef/IDs.lua

```diff
@@ -23,6 +23,7 @@ zones[xi.zone.ARRAPAGO_REEF] =
         STAGING_GATE_ILRUSI           = 7327, -- Ilrusi Atoll Staging Point.
         CANNOT_LEAVE                  = 7332, -- You cannot leave this area while in the possession of <keyitem>.
         RESPONSE                      = 7341, -- There is no response...
+        ASSAULT_START_OFFSET          = 7454, -- Max MP Down removed for <name>.
```

Comment:
This is a strange message to have as the offset for assaults. Is this because the actual assault messages have all of those huge ugly lists of areas/points etc. that are hard/ugly to capture in the comments for UpdateExtractor?

Reply:
I'm happy you pointed this out. 
A. It belongs in Ilrusi Atoll IDs.lua
B. All messaging for assaults in the assault instance zones begin from this line, correct. Checking quickly and there are around 250 messages for Ilrusi Atoll alone

PR'ing the adjustment to Ilrusi Atoll now

Reply:
adjustment corrected

Reply:
I didn't even catch the wrong zone :D

Reply:
(Makes sense to have this message as the starting offset message)